apps/wallet-mobile/src/features/Swap/common/PoolIcon/PoolIcon.tsx

```diff
@@ -20,6 +20,7 @@ const icons: Record<Swap.PoolProvider, FunctionComponent<{size?: number}>> = {
   muesliswap_v3: Icon.MuesliSwap,
   muesliswap_v4: Icon.MuesliSwap,
   minswap: Icon.MinSwap,
+  minswapv2: Icon.MinSwap,
```

Comment:
it looks like the convention here is _v

Reply:
is coming in this way from the API 🧐

Reply:
hmm, as `MINSWAPV2` ?

Reply:
maybe it should use Swap.Protocol, and not Swap.Provider.

Reply:
pool provider will die.